simulation_parameters/microbe_stage/organelles.json

```diff
@@ -1098,5 +1098,57 @@
     "RequiresNucleus": true,
     "EditorButtonGroup": "Multicellular",
     "EditorButtonOrder": 2
+  },
+  "ferroplast": {
+    "MPCost": 45,
+    "InitialComposition": {
+      "ammonia": 2,
+      "phosphates": 2
+    },
+    "Hexes": [
+      {
+        "r": 0,
+        "q": 0
+      },
+      {
+        "r": -1,
+        "q": 0
+      }
+    ],
+    "Processes": {
+      "ferrosynthesis": 1
+    },
+    "Components": {
+      "Storage": {
+        "Capacity": 1
+      }
+    },
+    "Enzymes": {
+      "rusticyanin": 1
+    },
+    "ProkaryoteChance": 0,
+    "ChanceToCreate": 0.5,
+    "Density": 1100,
+    "DisplayScene": "res://assets/models/organelles/Ferroplast.tscn",
+    "Name": "Ferroplast",
```

Comment:
I think this line is where the CI failure comes from, the name here needs to be a translation key.

Reply:
Yes, it copied that value into all the translation files, and then caused errors due to the lower case.